src/main/java/org/apache/commons/imaging/ImageFormat.java

```diff
@@ -33,6 +34,7 @@
      *
      * @return String extension
      */
-    String getExtension();
+    String[] getExtensions();
```

Comment:
Use `List<String>` instead of `String[]`? I generally find lists easier to work with, especially for streams.

Reply:
I started changing it, but later in the ImageParser implementations they are returning arrays for accepted extensions too. The commit started to grow a little too much, so I thought it'd be best to move it to a follow-up issue/discussion :+1:

Reply:
Tentative fix version set to alpha 3, or it might be done for 1.0 final release: https://issues.apache.org/jira/browse/IMAGING-322